### ISSUE TITLE
handling error

### ISSUE THREAD

**Issue by sikanhe** [artemeff/exredis#55]
Shouldn't we be using {:ok, vaue} and {:error, error} for responses as per elixir/erlang convention?

**Comment by artemeff** [comment]
@sikanhe yep, maybe in future releases I'll remove `|> elem(1)` from query API, thanks.

**Comment by seivan** [comment]
I second this as I'm forced to do    ` if foo == :undefined, do: foo = nil`
